spec.html

```diff
@@ -26405,7 +26502,9 @@ <h1>
             <emu-alg>
               1. If _state_.[[IsLoading]] is *false*, return ~unused~.
               1. If _moduleCompletion_ is a normal completion, then
-                1. Perform InnerModuleLoading(_state_, _moduleCompletion_.[[Value]]).
+                1. Let _loadType_ be ~recursive-load~.
+                1. If _phase_ is ~source~, set _loadType_ to ~single~.
```

Comment:
```suggestion
                1. Let _loadType_ be ~recursive-load~.
                1. If _phase_ is ~source~, let _loadType_ be ~single~; otherwise let _loadType_ be ~recursive-load~.
```